pages/linux/brctl.md

```diff
@@ -0,0 +1,23 @@
+# brctl
+
+> Ethernet bridge administration.
+
+- Create a new ethernet bridge interface:
+
+`sudo brctl add {{bridge_name}}`
+
+- Delete an existing ethernet bridge interface:
+
+`sudo brctl del {{bridge_name}}`
+
+- Add an interface to an existing bridge:
+
+`sudo brctl addif {{bridge_name}} {{interface_name}}`
+
+- Remove an interface from an existing bridge:
+
+`sudo brctl delif {{bridge_name}} {{interface_name}}`
+
+- Show a list with information about currently existing ethernet bridges:
```

Comment:
Let us make this the first example, since this is needed to get the bridge names.